spec/item_transferer_spec.rb

```diff
@@ -3,25 +3,36 @@
 describe ItemTransferer do
   before do
     @request_body = {
-      'holdingTransfers' =>
+      'itemTransfers' =>
       [
-        {'source'       => {'owningInstitutionBibId' => 'aBibId', 'owningInstitutionHoldingsId' => 'aHoldingsId'},
-         'destination' =>  {'owningInstitutionBibId' => '.destinationBibId', 'owningInstitutionHoldingsId' => 'aHoldingsId'}}
+        {'source'       => {
+          'owningInstitutionBibId' => 'aBibId',
+          'owningInstitutionHoldingsId' => 'aHoldingsId',
+          'owningInstitutionItemId' => 'anItemId'
+        },
+         'destination' =>  {
+           'owningInstitutionBibId' => '.destinationBibId',
```

Comment:
`bib_with_leading_dot()` only adds the "." if it doesn't already `start_with?('.')`.
`bib_with_leading_dot()` is a private method, since it doesn't quite make sense as a public interface. I could see testing its effects here as a good integration test.

Reply:
Actually - this is an integration tests of `bib_with_leading_dot`.

We construct the ItemTransferer like this, without a leading dot:

```ruby
    @item_transferer = ItemTransferer.new(
     api_key: 'fake_key',
     api_url: "http://example.com",
     barcode_to_attributes_mapping: {'1234' => {'owningInstitutionBibId' => 'aBibId', 'owningInstitutionHoldingsId' => 'aHoldingsId', 'owningInstitutionItemId' => 'anItemId'}},
     destination_bib_id: "destinationBibId"
   )
```

But assert the POST contains the leading dot...